cucumber/authenticators_azure/features/authn_azure_hosts.feature

```diff
@@ -0,0 +1,138 @@
+Feature: Azure Authenticator - Hosts can authenticate with Azure authenticator
+
+  In this feature we define an Azure authenticator in policy, define different
+  hosts and perform authentication with Conjur.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-azure/prod
+      body:
+      - !webservice
+
+      - !variable
+        id: provider-uri
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set Azure variables
+
+  Scenario: Host with user-assigned-identity annotation is authorized
+    And I have host "user-assigned-identity-app"
+    And I set subscription-id annotation to host "user-assigned-identity-app"
+    And I set resource-group annotation to host "user-assigned-identity-app"
+    And I set user-assigned-identity annotation to host "user-assigned-identity-app"
+    And I grant group "conjur/authn-azure/prod/apps" to host "user-assigned-identity-app"
+    And I fetch a user-assigned Azure access token from inside machine
+    When I authenticate via Azure with token as host "user-assigned-identity-app"
+    Then host "user-assigned-identity-app" has been authorized by Conjur
+
```

Comment:
Why in vanilla flow you dont verify something from log. for example : `Application identity validated`

Reply:
that's just a random message. i don't want to verify each message.

Reply:
maybe verify there is no error on logs? we dont know what happens in logs

Reply:
I don't think we need - or easily can - check that the logs don't contain error message. We have UTs for checking that if an error is raised then we fail the request so verifying a 200 response code is enough for that.